pages/managers.py

```diff
@@ -51,7 +51,7 @@ def on_site(self, site_id=None):
 
         :param site_id: specify the id of the site object to filter with.
         """
-        if settings.PAGE_USE_SITE_ID:
+        if global_settings.PAGE_USE_SITE_ID:
```

Comment:
This will force the user to setup the PAGE_USE_SITE_ID setting isn't it? Is this really what we want?